src/meta/meta_bulk_load_service.cpp

```diff
@@ -234,21 +246,31 @@ void bulk_load_service::create_app_bulk_load_dir(const std::string &app_name,
     ainfo.cluster_name = req.cluster_name;
     ainfo.file_provider_type = req.file_provider_type;
     ainfo.remote_root_path = req.remote_root_path;
-    blob value = dsn::json::json_forwarder<app_bulk_load_info>::encode(ainfo);
+    ainfo.is_ever_ingesting = false;
+    ainfo.bulk_load_err = ERR_OK;
 
```

Comment:
```
auto bulk_load_path = get_app_bulk_load_path(app_id);
  _meta_svc->get_meta_storage()->delete_node_recursively(
        bulk_load_path, [this, rpc, ainfo]() {
            ddebug_f("remove app({}) bulk load dir {} succeed", ainfo.app_name, bulk_load_path);
......
```

Reply:
"delete_node_recursively" need a r-value, so I can't pass bulk_load_path in to lambda.